cls/SourceControl/Git/Utils.cls

```diff
@@ -1784,6 +1793,15 @@ ClassMethod RunGitCommandWithInput(command As %String, inFile As %String = "", O
     set errStream = ##class(%Stream.FileCharacter).%OpenId(errLog,,.sc)
     set outStream = ##class(%Stream.FileCharacter).%OpenId(outLog,,.sc)
     set outStream.TranslateTable="UTF8"
+
+    if ((isCommit) && (returnCode = 128)){
+        set errStreamLine = errStream.ReadLine()
+        if (((errStreamLine = "Committer identity unknown") || (errStreamLine = "Author identity unknown"))) {
+            do errStream.Clear()
+            do errStream.WriteLine("Commit failed as user identity unknown."_$c(13,10)_$c(13,10)_"Go to settings to configure Git committer name and email."_$c(13,10))
```

Comment:
Let's be a bit more verbose, actually (here and elsewhere):
"Go to the Settings page"